jobmatch/src/main/resources/data.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Seed the Role database with possible user roles
+--
+MERGE INTO ROLE AS t USING (VALUES(1,'user'), (2, 'company')) AS vals(a,b)
```

Comment:
I'm not familiar with MERGE; is this similar to INSERT_OR_UPDATE? Could we replace this with a simple DELETE FROM ROLE; INSERT ...; INSERT ...;?